core/order.go

```diff
@@ -739,14 +809,35 @@ func validateCryptocurrencyOrderItem(item *pb.Order_Item) error {
 	return nil
 }
 
+func (n *OpenBazaarNode) getDivisibility(code string) uint32 {
+	dict := repo.LoadCurrencyDefinitions()
+	defn, err := dict.Lookup(code)
```

Comment:
`defn, err := dict.LoadCurrencyDefinitions().Lookup(code)` avoids unneeded allocations and is short enough for one line.